docroot/modules/custom/mass_fields/src/EntityAutocompleteMatcher.php

```diff
@@ -66,9 +66,10 @@ public function getMatches($target_type, $selection_handler, $selection_settings
       foreach ($entity_labels as $bundle => $values) {
         foreach ($values as $entity_id => $label) {
           if (!isset($entity_type_labels[$bundle])) {
-            $bundle_info = $handler->entityTypeBundleInfo->getBundleInfo($target_type)[$bundle];
-            if (isset($bundle_info)) {
-              $entity_type_label = $bundle_info['label'];
+            $node_type = $this->entityTypeManager->getStorage('node_type')
```

Comment:
`node_type` looks odd. i would expect an entity type like 'node', 'media', etc.

Reply:
@weitzman This is what we had in place before a recent round of New Relic-related fixes, and it seems to be the most direct way to retrieve bundle information for nodes without loading each entity, since using entityTypeBundleInfo doesn't appear to be available via the handler.